requirements/action.txt

```diff
@@ -1,4 +1,4 @@
-mmpycocotools
+# mmpycocotools
```

Comment:
Is it intended commenting out instead of removing `mmpycocotools`?

Reply:
i've removed all dependencies of the mmpycocotools from requirements for resolving some conflict while setting testenv but i'm not sure whether it could make any side effect or not. so i've left them with comment and wanted to let the task owner know what was changed on dependency.

Reply:
Thank you for kind explanation! @sungmanc do you know the history of `mmpycocotools`? 😮

Reply:
@JihwanEom , in my knowledge, mmpycocotools is not used anymore since we changed mmdetection as a latest version